### DISCUSSION TITLE
Change `/ess dump *` to `/ess dump all`

### DISCUSSION THREAD

**Discussion by JasonHorkles** [EssentialsX/Essentials#4515] [Ideas and Feature Suggestions] [answered]
Other ideas are make `/ess dump` give all info while `/ess dump partial` will give what the current normal dump provides cuz frankly nobody is really going to use `/ess dump` except people giving support in the Discord (which is usually from the helpers and staff anyways)

**Comment by Evidentsinger14** [comment]
+1 People tend to ignore the * in `/ess dump *` when generating the report. Pls <3

**Comment by JRoy** [comment, marked as answer]
https://github.com/EssentialsX/Essentials/commit/c85d772e1b709bb0d65f9ff7de10879029805cf0
